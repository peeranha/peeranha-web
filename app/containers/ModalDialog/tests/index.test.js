// import React from 'react';
// import { shallow } from 'enzyme';

// import { ModalDialog } from '../index';

describe('<ModalDialog />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
