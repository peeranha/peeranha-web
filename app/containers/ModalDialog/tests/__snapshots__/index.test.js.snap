// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalDialog /> render test 1`] = `
<ModalDialog__Box
  aria-hidden="true"
  aria-labelledby="myLargeModalLabel"
  className="modal fade"
  id="Object"
  role="dialog"
  tabIndex="-1"
>
  <div
    className="modal-dialog modal-dialog-centered"
  >
    <div
      className="modal-content"
    >
      <ModalWrapper>
        <div>
          Children
        </div>
      </ModalWrapper>
    </div>
  </div>
</ModalDialog__Box>
`;
